pyTigerGraph/pyTigerGraphQuery.py

```diff
@@ -663,7 +663,7 @@ def getQueryDescription(self, queryName: Optional[Union[str, list]] = "all"):
         if not res["error"]:
             if logger.level == logging.DEBUG:
                 logger.debug("exit: getQueryDescription")
-            return res["queries"]
+            return res["results"]["queries"]
```

Comment:
We could potentially raise an error if the response format doesn't conform to the expected:

```
{
    "results": {
         "queries": [...]
     }
}
```

As well. Or, we could add in some sort of unit testing with mocked responses for this.

Reply:
Yeah let's raise an error. Maybe we can do try just `["queries"]` if `["results"]["queries"]` fails, and then if both fail then we raise an exception

Reply:
Because I am not sure what the expected result is if a query doesn't have a description.

Reply:
Sounds good! I'll address this tomorrow.

Reply:
If a query doesn't have a description, it'll just... list the result as-is with no `"description"` portion for that parameter.

I put more details in the linked ticket for other cases too.

Sample:

```
{
    "error": false,
    "message": "Retrieve descriptions successfully.",
    "results": {
        "queries": [
            {
                "queryName": "search_any_vertex",
                "parameters": [
                    {
                        "paramName": "keyword"
                    }
                ]
            }
        ]
    }
}
```

Reply:
Good callout. This is good for now, we will merge here. Then we need to error check in CoPilot. Thanks again @jim-limprasert-tigergraph!